lib/fluent/plugin/out_fork.rb

```diff
@@ -62,9 +63,11 @@ def emit(tag, es, chain)
           end
         end
 
-        values.reject{ |value| value.to_s == '' }.each do |value|
+        values.reject{ |value| value.to_s == '' }.each_with_index do |value, i|
```

Comment:
For your purpose of this change, is it ok to count the index without empty values?